src/LoopDecorator.php

```diff
@@ -177,9 +177,9 @@ public function isTimerActive(TimerInterface $timer)
     public function nextTick(callable $listener)
     {
         $this->emit('nextTick', [$listener]);
-        return $this->loop->nextTick(function () use ($listener) {
+        return $this->loop->nextTick(function (LoopInterface $loop) use ($listener) {
             $this->emit('nextTickTick', [$listener]);
-            $listener();
+            $listener($loop);
```

Comment:
Can you change `$loop` to `$this` so the listener also gets the decorator and we catch all calls? Just found out, thanks you to :+1: , I made this mistake with the streams so I filed #2 .